bikeshare_app/static/css/login.css

```diff
@@ -88,4 +88,11 @@ label {
 
 .ui.form .inline.fields {
     justify-content: center;
-}
\ No newline at end of file
+}
+
+/*=================
+* Checkin Page
+===================*/
+#damageTextInput{
```

Comment:
One day I'll have the style tests set up to enforce a space between your id and brace... today is not that day.